Client/Frontend/Browser/GridTabViewController.swift

```diff
@@ -186,7 +186,9 @@ class GridTabViewController: UIViewController, TabTrayViewDelegate, Themeable {
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         self.view.layoutIfNeeded()
-        focusItem()
+        DispatchQueue.main.async {
+            self.focusItem()
+        }
```

Comment:
Would adding this in view did appear have a similar effect?

Also I know you didn't add it in this PR but the line above "self.view.layoutIfNeeded()" is redudant here because layoutSubviews is called during the initial lifecycle events regardless so you can probably remove that.

Reply:
If I put it in viewDidAppear(), there is a slight delay and it doesn't look as fluid.